orte/mca/oob/tcp/oob_tcp_component.c

```diff
@@ -410,6 +410,37 @@ static int tcp_component_register(void)
                                           &mca_oob_tcp_component.disable_ipv6_family);
 #endif
 
+    
+    mca_oob_tcp_component.keepalive_time = -1;
+    (void)mca_base_component_var_register(component, "keepalive_time",
+                                          "Idle time in seconds before starting to send keepalives (num <= 0 => disable keepalive)",
```

Comment:
nit: the `=>` looks odd next to the `<=`, might be better to replace with `-->` or `means`